R/print.R

```diff
@@ -442,6 +442,61 @@ print.bbi_nmboot_summary <- function(x, .digits = 3, .nrow = 10, ...) {
   }
 }
 
+
+#' @describeIn print_bbi Prints the `NM-TRAN` evaluation of a `bbi_nonmem_model`
+#'  object
+#' @export
+print.nmtran_process <- function(x, ...){
+
+  is_valid_print <- function(.x) {
+    if (!is.null(.x)) {
+      length(.x) != 0
+    } else {
+      FALSE
+    }
+  }
+
+  heading <- cli_h1
+  subheading <- cli_h2
+  bullet_list <- cat_bullet
+
+  status <- x[["status"]]
+  if (x[["status_val"]] == 0) {
+    status <- col_green(status)
+  } else {
+    status <- col_red(status)
+  }
+
+  nm_version <- x[["nonmem_version"]]
+  if(is.null(nm_version)) nm_version <- "unknown"
+
+  heading(cli::col_magenta("NM-TRAN Status"))
+  subheading(status)
+
+  heading("Absolute Model Path")
+  bullet_list(x[[ABS_MOD_PATH]])
+
+  heading("NM-TRAN Specifications")
+  cli::cli_bullets(
+    c(
+      "*" = "NM-TRAN Executable: {.path {x[['nmtran_exe']]}}",
+      "*" = "nmtran_presort: {cli::col_cyan(!is.null(x[['nmtran_presort_exe']]))}",
+      "*" = "NONMEM Version: {.val {nm_version}}"
```

Comment:
Mostly thinking aloud: I suppose this is redundant with the "NM-TRAN Executable" entry because, by definition, it's the name of the subdirectory under the parent directory containing the NONMEM installations (i.e. `/opt/NONMEM/` on Metworx).

Still it's probably a good idea to highlight the key subdirectory.

Reply:
Would could continue to store this information and just change what we print. I hear the redundancy argument, and maybe printing just the nonmem version would be sufficient